libraries/chain/webassembly/privileged.cpp

```diff
@@ -196,7 +196,7 @@ namespace eosio { namespace chain { namespace webassembly {
                                                                 .public_key{fc::crypto::blslib::bls_public_key{*pk}}});
       }
 
-      EOS_ASSERT( finpol.threshold > weight_sum / 2, wasm_execution_error, "Finalizer policy threshold cannot be met by finalizer weights" );
+      EOS_ASSERT( weight_sum >= finpol.threshold && finpol.threshold > weight_sum / 2, wasm_execution_error, "Finalizer policy threshold (${t}) cannot be met by finalizer weights (${w})", ("t", finpol.threshold)("w", weight_sum) );
```

Comment:
If we have 2 finalizers with `weight==1` each, that is a `weight_sum==2`. Isn't a threshold of 1 OK?
With the test as is: `finpol.threshold > weight_sum / 2`, threshold would have to be 2.

Reply:
@heifner and @arhag ?

Reply:
I believe threshold of 2 is desired for 2 finalizers. Otherwise the one finalizer is all that would be needed to finalize the block.

Reply:
It is not only 2. Assuming weights of 1, the current check requires a minimum threshold of 3 for 4 finalizers, 4 for 6, ...

Reply:
The hotstuff paper uses `2f + 1` as threshold where `n = 3f + 1`. Our version of minimum `weight_sum/2` looks reasonable.

Reply:
`threshold > weight_sum / 2` is correct.